docker/px4-dev/scripts/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Add local user
+# Either use the LOCAL_USER_ID if passed in at runtime or
+# fallback
+
+USER_ID=${LOCAL_USER_ID:-9001}
+
+echo "Starting with UID : $USER_ID"
+useradd --shell /bin/bash -u $USER_ID -o -c "" -m user
```

Comment:
Yes, I'm able to enter the same container multiple times and it doesn't create a new user.